packages/yew/tests/suspense.rs

```diff
@@ -593,3 +593,44 @@ async fn effects_not_run_when_suspended() {
     );
     assert_eq!(*counter.borrow(), 4); // effects ran 4 times.
 }
+
+#[wasm_bindgen_test]
+async fn use_suspending_future_works() {
+    #[function_component(Content)]
+    fn content() -> HtmlResult {
+        let _sleep_handle = use_future(async move {
+            TimeoutFuture::new(50).await;
```

Comment:
Could you manually resolve a `Future`? Sleeping for 50ms is often, but not always, guaranteed to be longer than 10ms (where you test that it's still waiting). I'd rather have a manually triggered future here with a reliable resolved/pending state.

Reply:
The other tests do this and I copied it from there. It has never become a problem. If it ends up being a problem, we can find a solution in the future